manage/installing/installation.rst

```diff
@@ -125,11 +125,11 @@ Download the latest Plone unified installer
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Download from `the plone.org download page <https://plone.org/download>`_ to your server using wget command. Curl also works.
```

Comment:
I like the overall direction of yopur idea, but I would then make it more generic, like so:

- https://plone.org/download -> overview (folder_summary_view or something like that) OR redirect to latest (both ways each has it pros and cons)
- https://plone.org/download/latest -> always redirects to most current release
- https://plone.org/download/5-latest -> the link to be used in the 5.x series documentation
- https://plone.org/download/4-latest -> the link to be used in the 4.x series documentation
- https://plone.org/download/3-latest -> the link to be used in the 3.x series documentation (only if really still needed)
- https://plone.org/download/old -> https://launchpad.net/plone/+download